httpwrapper/httpwrapper.go

```diff
@@ -40,8 +41,16 @@ func is2xxSuccessful(statusCode int) bool {
 	return statusCode >= 200 && statusCode <= 299
 }
 
+func printAction(method string, url string) {
+	if viper.IsSet("verbose") {
+		_, _ = fmt.Fprintf(os.Stderr, "HTTP request: method = '%s', url = '%s'\n", method, url)
```

Comment:
If we're just printing to the standard output, why not just use `fmt.Printf`?

Reply:
We're _not_ printing to standard output but standard error. I can pipe the result of the CLI to other tools for JSON processing (or YAML if we allow that) without running the risk of getting logging passed to the next stage. This is common practice for (Unix) command-line programs. The basic tenet of the Unix command-line is to build small but powerful tools, that do one thing and do it well and fast. These are can then be combined as needed using e.g. pipes. By creating "master" tools that attempt to combine all functionality a user would ever want, you require intimate knowledge of all fields. Divide and conquer prevents that.

Reply:
Hmm, but why `Stderr`? Isn't that reserved for...errors?
And isn't this a bit of an early "optimization"? I like that you put the logger in the `httpWrapper`, so I'm thinking that if we need to print to a non-default output, we can just change it in one place.

Reply:
```
$ ./axon-server-cli cluster list | jq -r '.[].name'

node-1
node-2
node-3
$
```
If the logging goes to stdout, I get text mixed with JSON data, and I cannot do anything useful with the output.

Reply:
Is outputing to a file currently a requirement?

Reply:
I am fine about using the `Stderr` here and probably should use it in other places as well.

Reply:
You guys are right. If we're doing logging, we should output it there anyway, since all the crashes/panics are outputted to `Stderr` as well. Makes sense!